docs/samples/cabundle/README.md

```diff
@@ -0,0 +1,100 @@
+# KServe with Self Signed Certificate Model Registry
```

Comment:
It is probably better to move these docs to the website for more visibility: https://github.com/kserve/website

Reply:
yes good idea